src/python/WMCore/MicroService/MSPileup/MSPileupTasks.py

```diff
@@ -173,18 +184,11 @@ def monitoringTask(doc, spec):
     # get list of existent rule ids
     pname = doc['pileupName']
     kwargs = {'scope': 'cms', 'account': rucioAccount}
-    if isinstance(rucioClient, Client):
```

Comment:
I had understood that we would keep both implementations around until we are sure that both implementations are equally functional. Does it mean you are happy with the current tests you have done so far using Rucio wrapper module?